ui/i18n/src/main/res/values/strings.xml

```diff
@@ -37,6 +37,7 @@
     <string name="swipe_left">Swipe Left</string>
     <string name="enable_swipeactions">Enable Swipe Actions for this Screen</string>
     <string name="change_setting">Change</string>
+    <string name="do_not_show_remove_inbox_prompt">Never ask again.</string>
```

Comment:
We already have a string for this on the "queue" screen: `checkbox_do_not_show_again`